earth_enterprise/src/third_party/apache2/apache-ge-httpd.conf.in-cgi-bin_admin.patch

```diff
@@ -1,6 +1,7 @@
 diff -uNr httpd-2.2.29.orig/docs/conf/httpd.conf.in httpd-2.2.29/docs/conf/httpd.conf.in
---- httpd-2.2.29.orig/docs/conf/httpd.conf.in	2015-07-30 17:09:30.644259782 -0700
-+++ httpd-2.4.39/docs/conf/httpd.conf.in	2015-07-30 17:09:52.219686038 -0700
++++ httpd-2.4.39.orig/docs/conf/httpd.conf.in	2015-07-30 17:09:52.219686038 -0700
```

Comment:
Should this be `httpd-2.4.46`?

Reply:
Yeah, it should.

Reply:
The other patch files have the same issue. It may not affect how they work, though.

Reply:
Yeah, I started to change all of them, but I noticed they were still being applied and they weren't even updated from 2.2 in some cases, so I researched how `patch` works and found out it uses fuzzy matching to handle cases like this where a version changes and line numbers and version strings don't exactly match up.

I thought I reverted all of them but this one got through.